ReduxSimple/ReduxStore.cs

```diff
@@ -34,10 +34,16 @@ public IObservable<TState> ObserveState()
             return _stateSubject.AsObservable().DistinctUntilChanged();
         }
 
+        public IObservable<TPartial> ObserveState<TPartial>(Func<TState, TPartial> selector)
+        {
+            return _stateSubject.Select(selector).DistinctUntilChanged();
+        }
+
         public IObservable<object> ObserveAction()
         {
             return _actionSubject.AsObservable();
         }
+
```

Comment:
Overloaded methods (with the same name) should be close to one another. The same should be applied for `ObserveState`.

Reply:
Will do.